custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml

```diff
@@ -97,7 +97,7 @@ card_media_player:
                 ulm_popup_media_player_entity: "[[[ return entity.entity_id ]]]"
         icon: |
           [[[
-            var icon = "mdi:speaker";
+            var icon = states[entity.entity_id].attributes.icon;
```

Comment:
This is useless because 3 "if" below you already have the case but with the change we are loosing the default value

Reply:
This line is for custom players and it means that it will use the entity/attribute icon automatically instead of a constant value"mdi:speaker". By default, HomeAssistant also set it to "mdi:speaker" so no issue with that. It just gives the option to be dynamic.

Reply:
> I'm not sure everybody want this behaviour. I personal use the card in a small size and adding `media_artist` won't fit on my case. Maybe you can add it as a variable

Will do try to make it an option :)